hermes/probe/delete/delete_test.go

```diff
@@ -170,7 +167,7 @@ func TestDeleteRandomFile(t *testing.T) {
 			t.Errorf("deleteRandomFile failed, expected object to be deleted, got object found.")
 		}
 		if err != nil {
-			t.Errorf("deleteRandomFile failed, expected %v, got %v", nil, err)
+			t.Fatalf("deleteRandomFile failed, expected %v, got %v", nil, err)
```

Comment:
There's no expectation here, so the "expected" part is not required.
Use this message format for these kinds of failures `t.Fatalf("<operation desc> failed: %v", err)`.

Reply:
Okay, thanks. I've made this change.